newscoop/application/configs/symfony/security.yml

```diff
@@ -83,15 +83,15 @@ security:
             provider: frontend_db
             form_login:
                 login_path:  /auth
-                check_path:  /auth/login_check
+                check_path:  /auth
```

Comment:
It can, but only if someone new that he should use `login_check`. By default all our forms and templates always use /auth or "" (empty string) for action parameter - so at end  `login_check` was not used. So this change covers all of them with Symfony firewall. We will need add this to release notes.